Libraries/CMSIS/Device/Maxim/MAX32672/Include/max32672.h

```diff
@@ -278,13 +278,18 @@ typedef enum {
 #define MXC_WDT1 ((mxc_wdt_regs_t *)MXC_BASE_WDT1)
 
 /******************************************************************************/
-/*                                                                   AES Keys */
-#define MXC_BASE_AESKEYS ((uint32_t)0x40205000UL)
-#define MXC_AESKEYS ((mxc_aeskeys_regs_t *)MXC_BASE_AESKEYS)
+/*                                                   System and User AES Keys */
+#define MXC_BASE_SYS_AESKEYS ((uint32_t)0x40205000UL)
+#define MXC_SYS_AESKEYS ((mxc_sys_aeskeys_regs_t *)MXC_BASE_SYS_AESKEYS)
+
+#define MXC_BASE_USR_AESKEYS ((uint32_t)0x40005000UL)
+#define MXC_USR_AESKEYS ((mxc_usr_aeskeys_regs_t *)MXC_BASE_USR_AESKEYS)
 
 // DEPRECATED(1-10-2023): Scheduled for removal.
```

Comment:
Check if `@deprecated` works on macros as well?

Reply:
It will throw an error if the c code sees the deprecated annotation. Will Doxygen know the macros are deprecated if you add a comment `// @deprecated` within the same line for each macro?

Reply:
You should be able to add a Doxygen header for a preprocessor definition.  Though you may need the changes from #521 that enable Doxygen output for static members.  See [here](https://github.com/Jake-Carter/msdk/blob/0240928ae0077cb4651bbe52a4c526f63a71a2d0/Libraries/PeriphDrivers/Include/MAX32670/mxc_sys.h#L226) for an example where I did a doxygen header for a macro.

Ex:
```C
/*
 * @deprecated (description...)
 */
#define MXC_BASE_AESKEY MXC_BASE_AESKEYS
```

If you want to do it inline I think you have to do something like what we did for the enums.  Haven't tried this myself

```C
#define MXC_BASE_AESKEY MXC_BASE_AESKEYS /**< @deprecated (description) */
```